.changeset/gorgeous-garlics-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@builder.io/qwik': major
```

Comment:
This should normally be `patch`, but in this case it's a fix for the docs so no changeset is needed, can you remove this file?